pensyve-mcp-gateway/scripts/test-model-bundle.sh

```
@@ -0,0 +1,178 @@
+        cat "${output_file}" >&2
+        return 1
+    fi
+    if [[ -n "${expected_two}" ]] \
+        && ! grep -F -- "${expected_two}" "${output_file}" >/dev/null; then
+        echo "verification failure did not name ${expected_two}" >&2
+        cat "${output_file}" >&2
+        return 1
+    fi
+}
+
+readonly TEST_ROOT="$(mktemp -d "$(dirname -- "${FIXTURE_ROOT}")/.pensyve-model-test.[literal].XXXXXX")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)